backend/core/src/units/entities/unit.entity.ts

```diff
@@ -156,6 +160,27 @@ class Unit {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
   bmrProgramChart?: boolean | null
+
+  @ManyToOne(() => UnitType, { eager: true, nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => UnitType)
+  unitTypeRef?: UnitType | null
```

Comment:
Shouldn't this be unitType, not unitTypeRef?

Reply:
Also, `unitType` is still defined here as a string, which I don't think we want anymore with this update. With that in mind, wouldn't we also want a script to create a the unit types based off of these strings and assign them accordingly?